src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapPartialResultsProcessor.cs

```diff
@@ -345,7 +345,8 @@ internal PartialResultsRetriever(ManualResetEvent eventHandle, LdapPartialResult
             // Start the thread.
             var thread = new Thread(new ThreadStart(ThreadRoutine))
             {
-                IsBackground = true
+                IsBackground = true,
+                Name = ".NET Ldap Results"
```

Comment:
Ldap => LDAP ?

Also, the "Results" part is strange.  Wouldn't Retriever or Searcher or something like that be better?